src/master/develop/management.md

```diff
@@ -250,11 +264,218 @@ If you're using another development language, please refer to the [immugw](/mast
 
 ::::
 
+### Database listing
+
+This example shows how to list existent databases using `DatabaseListV2` method.
+
+:::: tabs
+
+::: tab Go
+
+```go
+	client, err := immudb.NewClient()
+	if err != nil {
+    	log.Fatal(err)
+	}
+
+	ctx := context.Background()
+	
+	err = client.OpenSession(ctx, []byte(`immudb`), []byte(`immudb`), "defaultdb")
+	if err != nil {
+    	log.Fatal(err)
+	}
+
+	defer client.CloseSession(ctx)
+
+	res, err := client.DatabaseListV2(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, db := range res.Databases {
+		fmt.Printf("database: %s, loaded: %v\r\n", db.Name, db.Loaded)
+	}
+```
+:::
+
+::: tab Java
+```java
+	List<String> dbs = immuClient.databases();
```

Comment:
This uses old API, maybe we should wait for the SDK to be updated before adding java example ?

Reply:
while it's not the latest API, it will list database names at least